include/suit_types.h

```diff
@@ -11,7 +11,7 @@
 #include <string.h>
 #include "zcbor_common.h"
 #include "manifest_types.h"
```

Comment:
Could you check if we can remove the lines 12-13?
In such case, we will be able to include this file without dependencies to ZCBOR-generated files! :+1: